### DISCUSSION TITLE
请问如何配置图床远程api？

### DISCUSSION THREAD

**Discussion by sucooer** [Qexo/Qexo#573] [Q&A]
图床是cloudflare+telegram搭建的，我用sharex和piclist都能够上传，相同的配置在qexo上却上传失败，报错：`SONDecodeError('Expecting value: line 1 column 1 (char 0)')`

**Comment by am-abudu** [comment]
Json格式不标准。请不要使用任何的无关空格或单引号

**Reply by sucooer** [reply]
> Json格式不标准。请不要使用任何的无关空格或单引号

没有空格和单引号
`POST参数名`是`file`,`JSON路径`是`0.src`,应该是没问题的吧？

**Reply by am-abudu** [reply]
可以先尝试将JSON路径留空
